tests/sentry/workflow_engine/handlers/condition/test_age_comparison_handler.py

```diff
@@ -47,16 +50,30 @@ def test_dual_write(self):
         assert dc.type == self.condition
         assert dc.comparison == {
             "comparison_type": AgeComparisonType.OLDER,
-            "value": "10",
+            "value": 10,
             "time": "hour",
         }
         assert dc.condition_result is True
         assert dc.condition_group == dcg
 
+    def test_json_schema(self):
+        with pytest.raises(ValidationError):
+            self.dc.comparison.update({"time": "asdf"})
```

Comment:
the validation error is raised on save, right? maybe you can put the update outside of the pytest raise here to make that more clear